tests/Performance/LazyLoading/ProxyInitializationTimeBench.php

```diff
@@ -1,71 +0,0 @@
-<?php
-
-declare(strict_types=1);
-
-namespace Doctrine\Performance\LazyLoading;
-
-use Doctrine\ORM\Proxy\InternalProxy as Proxy;
-use Doctrine\Performance\EntityManagerFactory;
-use Doctrine\Performance\Mock\NonProxyLoadingEntityManager;
-use Doctrine\Tests\Models\CMS\CmsEmployee;
-use Doctrine\Tests\Models\CMS\CmsUser;
-
-/** @BeforeMethods({"init"}) */
-final class ProxyInitializationTimeBench
-{
-    /** @var Proxy[] */
-    private array|null $cmsUsers = null;
-
-    /** @var Proxy[] */
-    private array|null $cmsEmployees = null;
-
-    /** @var Proxy[] */
-    private array|null $initializedUsers = null;
-
-    /** @var Proxy[] */
-    private array|null $initializedEmployees = null;
-
-    public function init(): void
```

Comment:
This bench is effectively verifying the time it takes to load proxies (the callbacks): it should use a different API (compared to `__load`, but it should remain

Reply:
Done